test/OpenTelemetry.Exporter.Geneva.Tests/GenevaMetricExporterTests.cs

```diff
@@ -1264,125 +1026,71 @@ private static void AssertExemplarFilteredTagSerialization(Exemplar expectedExem
         }
     }
 
-    private static UserdataV2 GetSerializedData(Metric metric, GenevaMetricExporter exporter)
+    private static UserdataV2 GetSerializedData((MetricEventType EventType, byte[] Body) singleEvent)
     {
-        var metricType = metric.MetricType;
-        var metricPointsEnumerator = metric.GetMetricPoints().GetEnumerator();
-        metricPointsEnumerator.MoveNext();
-        var metricPoint = metricPointsEnumerator.Current;
+        var stream = new KaitaiStream(singleEvent.Body);
+        var data = new MetricsContract(stream);
+        Assert.Equal((ushort)MetricEventType.TLV, data.EventId);
+        return data.Body as UserdataV2;
+    }
 
-        var exemplars = metricPoint.GetExemplars();
-        UserdataV2 result = null;
+    private static byte[] TransportCopy(byte[] src, int size)
+    {
+        int headerSize;
+        unsafe
+        {
+            headerSize = sizeof(BinaryHeader);
+        }
 
-        if (metricType == MetricType.LongSum)
+        var result = new byte[size + headerSize];
+        Array.Copy(src, result, size + headerSize);
```

Comment:
I wonder if it's a bug when the exporter calls IMetricDataTransport.Send the _size_ parameter doesn't include binary header size.